### PR TITLE
Change logLevel property to reddeer.logLevel (followup on #1105)

### PR DESCRIPTION
**Pull request by ljelinkova**

### CODE REVIEW

File: plugins/org.jboss.reddeer.ui/src/org/jboss/reddeer/eclipse/ui/launcher/RedDeerLauncherProperties.java [hunk 1]
```diff
@@ -46,11 +46,12 @@ public RedDeerLauncherProperties(RedDeerProperties property) {
...
 				String argName = key.replace(RedDeerLauncherProperties.ATTRIBUTE_PREFIX, "");
 				try{
-					RedDeerProperties rdProperty = RedDeerProperties.getByName(argName);
+					RedDeerProperties rdProperty = getByName(argName);
 					RedDeerLauncherProperties property = new RedDeerLauncherProperties(rdProperty);
 					String argValue = configuration.getAttribute(key, property.getProperty().getValue());
 					property.setCurrentValue(argValue);
-					property.setDoubleDefined(configuration);;
+					property.setDoubleDefined(configuration);
+					System.out.println("Found property: " + property.getProperty().getName() + " with value " + property.currentValue);
```
Comment (vpakan) [inline plugins/org.jboss.reddeer.ui/src/org/jboss/reddeer/eclipse/ui/launcher/RedDeerLauncherProperties.java]: Please remove this line

File: plugins/org.jboss.reddeer.ui/src/org/jboss/reddeer/eclipse/ui/launcher/RedDeerLauncherProperties.java [hunk 2]
```diff
@@ -61,6 +62,16 @@ public RedDeerLauncherProperties(RedDeerProperties property) {
 
 		return properties;
 	}
+	
+	private static RedDeerProperties getByName(String name){
+		RedDeerProperties rdProperty;
+		try {
```
Comment (vpakan) [inline plugins/org.jboss.reddeer.ui/src/org/jboss/reddeer/eclipse/ui/launcher/RedDeerLauncherProperties.java]: Do not use try catch please instead use if (!name.startsWith("reddeer.") ...

**Comment by vpakan** [comment]
Please create issue which will update documentation of RedDeer properties

**Comment by ljelinkova** [comment]
Created documentation issue #1149

**Comment by ljelinkova** [comment]
Retest this

**Pull request merged**